_layouts/doc_page.html

```diff
@@ -52,7 +52,12 @@
 
     <script>
     $(function() {
-      $(".toc").load("/docs/{{ version }}/toc.html");
+      $toc = $(".toc");
+      $toc.load("/docs/{{ version }}/toc.html", function () {
+        $toc.find('a').each(function(idx, el) {
+          el.href = el.href.replace('replace-with-version', '{{ version }}');
```

Comment:
@xvrl would you mind doing a PR with that? I am not really sure what you're talking about and I suspect that's the easiest way to explain it

I'm open to anything that works